sys/mobileapps.yaml

```diff
@@ -39,6 +39,8 @@ paths:
             request:
               method: get
               uri: '{+options.host}/{domain}/v1/page/mobile-sections/{title}'
+              headers:
+                accept-encoding: null
```

Comment:
> Actually, should we set this more generally for all backend service requests?

I would like to deploy this one first and then see how it affects performance of this endpoint before making a more generic decision.